fury/actor.py

```diff
@@ -195,3 +195,192 @@ def box(
     obj.prim_count = prim_count
 
     return obj
+
+
+def cylinder(
+    centers,
+    colors,
```

Comment:
let's put a default value to `colors` and avoid to make it mandatory as discussed with @m-agour